confignode/src/main/java/org/apache/iotdb/confignode/manager/load/heartbeat/ConfigNodeHeartbeatCache.java

```diff
@@ -60,23 +60,15 @@ public boolean updateLoadStatistic() {
       }
     }
 
-    NodeStatus originStatus;
-    switch (status) {
-      case Running:
-        originStatus = NodeStatus.Running;
-        break;
-      case Unknown:
-      default:
-        originStatus = NodeStatus.Unknown;
-    }
+    String originStatus = status.getStatus();
```

Comment:
what's the difference between `Unknown` and `Error`?
`Unknown` -> `Error` also need to update?

Reply:
Fixed. We only need to broadcast the latest RegionRouteMap when Running changes to another state or when other states change to Running.